pkg/services/sqlstore/migrator/migrator.go

```diff
@@ -168,6 +171,15 @@ func (mg *Migrator) exec(m Migration, sess *xorm.Session) error {
 	return nil
 }
 
+func (mg *Migrator) ClearMigrationEntry(id string) {
+	sess := mg.x.NewSession()
+	defer sess.Close()
+	_, err := sess.SQL(`DELETE from migration_log where migration_id == 'move dashboard alerts to unified alerting'`).Query()
+	if err != nil {
+		panic(err)
```

Comment:
Not sure I want to change the change of `AddMigration` > `AddMigrations` to include error so living with panic. Maybe better to os exit instead of panicing if not passing error back.